package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@zenika/sensei",
   "version": "1.0.0",
   "scripts": {
-    "postinstall": "npm install --prefix=src/pdf",
```

Comment:
This is what is causing @zigarn's problem. There should be no change to package-lock.json either.

Reply:
Ok, I will revert those changes.